### ISSUE TITLE
Verify Rumble works correctly

### ISSUE THREAD

**Issue by C0rn3j** [C0rn3j/sc-controller#52]
Does not seem like I could trigger it on a PS4 controller.

https://hardwaretester.com/gamepad

Reported back in Ryochan's repo here - https://github.com/Ryochan7/sc-controller/issues/105

**Comment by DiggyWombat** [comment]
Hey @C0rn3j. Have you (or the upstream dev) managed to get anywhere with rumble support on PlayStation controllers? I have a DualSense PS5 controller that does not correctly work with xinput rumble commands. Strangely - the 'feedback' functionality does work, so if I set the controller to rumble on moving the right analogue stick for example, then we will get a brief rumble.

**Comment by C0rn3j** [comment]
Nope, I would like to get it working though, just haven't had the time to look at it, help welcome.

**Comment by C0rn3j** [comment]
DS4 was missing rumble support(feedback) - Fixed in https://github.com/C0rn3j/sc-controller/commit/02aaeed2a004e4cb78e2694ea6c744652cd80d5b
